tools/generate-helpers/pg-table-bindings/resource_type.go

```diff
@@ -13,19 +13,16 @@ type ResourceType int
 const (
 	unknown ResourceType = iota
 	joinTable
-	permissionChecker
+	unused1 // was permission checker
```

Comment:
do we need this? I think it's an internal enum that is not stored anywhere

Reply:
Looking closer, we probably don't. I'll remove it.